kuma/attachments/feeds.py

```diff
@@ -10,15 +10,19 @@ class AttachmentsFeed(DocumentsFeed):
     subtitle = _("Recent revisions to MDN file attachments")
 
     def items(self):
-        return AttachmentRevision.objects.order_by('-created')[:50]
+        return (AttachmentRevision.objects.prefetch_related('creator',
+                                                            'attachment')
+                                          .order_by('-created')[:50])
```

Comment:
Would `select_related` be more efficient here, since `creator` and `attachment` are both `ForeignKey` relations? `select_related` would get them in the same query with a SQL join, rather than `prefetch_related` doing separate lookups for each and joining in Python?